### PR TITLE
Bump SDK version to 1.8.0

### PR DESCRIPTION
**Pull request by eskaur**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -29,13 +29,10 @@ stages:
...
-# Currently only newest_zivid_version is supported. Azure Pipelines requires at least one job under
-# jobs, so parameters.zivid_versions cannot be empty. Re-enable this when we support >0 old versions.
-#  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
-#    jobs:
-#    - template: continuous-integration/azure-pipelines/templates/build_and_test.yml
-#      parameters:
-#        zivid_versions: []
-#        ros_distros: ['ros:melodic-ros-base-bionic', 'ros:kinetic-ros-base-xenial']
-#        compilers: ['g++-7']
+  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
```
Comment (apartridge) [inline azure-pipelines.yml]: This needs to remain commented out when we only support 1 SDK

Comment (eskaur) [reply azure-pipelines.yml]: Fixed

File: azure-pipelines.yml [hunk 2]
```diff
@@ -38,4 +38,4 @@ stages:
 #      parameters:
 #        zivid_versions: []
 #        ros_distros: ['ros:melodic-ros-base-bionic', 'ros:kinetic-ros-base-xenial']
-#        compilers: ['g++-7']
+#        compilers: ['g++-7']
```
Comment (apartridge) [inline azure-pipelines.yml]: can you re-add newline here?

Comment (eskaur) [reply azure-pipelines.yml]: Sorry about that rush-job. Fixed now :)

**Pull request merged**